src/components/CityCategoryContainer/CityCategory/CityCategory.js

```diff
@@ -23,6 +24,7 @@ const CityCategory = props => (
       cities={props.cities}
       city={props.city}
       indicatorSubCategory={getSubCategoryOfHeroIndicator(props.category)}
+      indicatorColorName={getColorOfHeroIndicator(props.category)}
```

Comment:
Same as above. If `getSubCategoryOfHeroIndicator` returned the whole sub-category, this file would go unchanged.